ReactQt/runtime/src/layout/flexbox.cpp

```diff
@@ -155,10 +157,23 @@ void FlexboxPrivate::updatePropertiesForControlsTree(YGNodeRef node) {
 void FlexboxPrivate::updatePropertiesForControl(YGNodeRef node) {
     auto qmlControl = static_cast<FlexboxPrivate*>(YGNodeGetContext(node))->m_control;
 
+    bool emitLayoutUpdated = false;
```

Comment:
If this improves readability, we can write this statement as:
`bool emitLayoutUpdated = YGNodeLayoutGetLeft(node) != qmlControl->x() || ...`

Reply:
thanks!